package.json

```diff
@@ -295,7 +295,7 @@
     "@metamask/snaps-rpc-methods": "^5.0.0",
     "@metamask/snaps-sdk": "^1.4.0",
     "@metamask/snaps-utils": "^5.2.0",
-    "@metamask/transaction-controller": "^21.1.0",
+    "@metamask/transaction-controller": "^22.0.0",
```

Comment:
For audit sake, I assume our fix didn't warrant breaking changes, but this was just the timing of the release?

Reply:
It was all about timing, our fix was just a minor change. Thread of bc 
 discussion https://consensys.slack.com/archives/C01V1L10W2E/p1707823445055439